libraries/stdlib/samples/test/samples/aggregates/aggregates.kt

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright 2010-2017 JetBrains s.r.o.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package samples.aggregates
+
+import samples.Sample
+import kotlin.test.assertFalse
+import kotlin.test.assertTrue
+
+
+class Aggregates {
+    @Sample
+    fun all() {
+        val isEven: (Int) -> Boolean = { it % 2 == 0 }
+        val zeroToTen = 0..10
+        assertFalse {
+            zeroToTen.all(isEven)
```

Comment:
Though we can definitely use `all`/`any`/`none` with a callable reference, the most common usage is with a lambda function: `.all { isEven(it) }`. Probably, we can show both options.